examples/ecommerce-netlify-functions/test/addToCart.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { describe, it, before, after } = require('mocha');
+const assert = require('assert');
+const { handler: addToCart } = require('../functions/addToCart');
+const mongoose = require('mongoose');
+
+
+
+describe('Add to Cart', function() {
+  before(async() => {
+    await mongoose.connect('mongodb://localhost:27017/netlify');
+  });
+
+  after(async() => {
+    await mongoose.disconnect();
+  });
+  it('Should create a cart and add a product to the cart.', async function() {
+    const params = {
+      body: {
+        cartId: null,
+        product: [
+          { productId: '629e5f3686d82fc73b95b396', quantity: 2 },
```

Comment:
Please create fixtures for tests similar to how we do in the core-api repo. Running tests on seed data isn't good, because you need to remember to seed the data before every test. Also, you don't get a clean database on every test, which means changes from one test can cause another test to fail.